README.rst

```diff
@@ -4,14 +4,111 @@
 .. image:: https://travis-ci.org/camptocamp/marabunta.svg?branch=master
     :target: https://travis-ci.org/camptocamp/marabunta
 
-Marabunta is a name given to the migration of the legionary ants or to the ants
-themselves. Restless, they eat and digest everything in their way.
+Marabunta is used to provide an easy way to provide updates to Odoo which can be created fast and run easily. It also allows to differentiate between different environment to provide for instance demodata.
 
-This tool aims to run migrations for Odoo versions as efficiently as a
-Marabunta migration.
 
-It loads migration instructions from a YAML file and run the operations if
-required.
+Usage
+=====
+After installing marabunta will be available as a console command. To run properly it requires a migration file, which defines what should be updated/executed, and of course stuff like database pw. Most of these also can also be provided by an envvar and will be discussed in the options section.
+
+It is very much recomended to configure it, so that marabunta is ran automatically if odoo is started.
+For instance adding it to your docker entrypoint.
+
+Features
+========
+
+* backup: Marabunta allows for a backup command to be executed before the migration.
+* addon upgrades: Marabunta is able to install or upgrade odoo addons.
+* addon uninstall: Addons which were previously installed can be removed with marabunta.
+* operations: Allows to execute commands before or after upgrading modules.
+* modes: Modes allow the user to execute commands only on a certain environment. i.e. creation of demodata on a dev system.
+
+Versioning systems
+------------------
+Currently Marabunta allows for two different Versioning systems:
+The classic Major.Minor.Bugfix and the Five digits long versions for OdooMajor.OdooMinor.Major.Minor.Bugfix.
+Although the first marabunta version should always be called setup. This is a standin for all the versions till the release, to make it easier to keep marabunta versions and tags in sync.
```

Comment:
This one probably needs a rewording: "This is a standin for all the versions till the release, to make it easier to keep marabunta versions and tags in sync.". I'm not sure to understand it :)

Reply:
The 1st version must be [underlined] "setup", as you define here the initial setup of your instance (read more about the rationale here [link to changelog])